src/styles/date-selector/index.css

```diff
@@ -3,32 +3,59 @@
 @import "../colors.css";
 @import "./properties.css";
 
+.dateselector {
+  background-color: var(--date-selector-sidebar-background);
+
```

Comment:
1 - Wrong width
It could be void using some `inline`  display, like the following:
```suggestion
  display: inline-flex;
```
```suggestion
  display: inline-flex;
```
![screen shot 2019-02-19 at 14 33 22 2](https://user-images.githubusercontent.com/1773766/53035350-f93d8800-3453-11e9-93d2-86fc836b34e4.png)